deployment/clowdapp.yaml

```diff
@@ -575,9 +578,12 @@ objects:
         - name: SENTRY_TRACES_SAMPLE_RATE
           value: ${SENTRY_TRACES_SAMPLE_RATE}
         - name: CELERY_ENABLE_SENTRY
-          value: ${CELERY_ENABLE_SENTRY}
+          value: ${LISTENER_ENABLE_SENTRY}
         - name: CELERY_SENTRY_DSN
```

Comment:
`CELERY_*` sentry env vars are only used in the celery.py config file, so they'll only ever get used by the beat and worker. As the listener is a django management command, you'll want to populate the `DJANGO_*` sentry env vars that are used in prod.py with the listener DSN and other values.

Reply:
@kat commit: https://github.com/cloudigrade/cloudigrade/pull/989/commits/2f1a71737848816d00c9fd4de6e51d34821576df pushed. I `think` that takes care of it. Let me know if I missed anything else.  Thanks.